corehq/apps/case_importer/views.py

```diff
@@ -403,6 +404,19 @@ def excel_commit(request, domain):
         case_upload.check_file()
     except ImporterError as e:
         return render_error(request, domain, get_importer_error_message(e))
+    num_cases = 0
+    worksheet_titles = _get_workbook_sheet_names(case_upload)
+    for i in range(len(worksheet_titles)):
+        with case_upload.get_spreadsheet(i) as spreadsheet:
+            num_cases += spreadsheet.max_row - 1
+
+    if not request.GET.get('confirm', False):
```

Comment:
should the user have to confirm their decision every time now? I think the original ticket asked for flagging abnormalities, rather than adding another step in the process. If this were all `HTMX` another confirm step would feel pretty seamless... However, HQ isn't ready for that yet and a full page reload is quite a cumbersome addition for most users.

Reply:
Also, I noticed that this view is `require_POST` yet the check looks at the request's `GET` parameters because of [this line](https://github.com/dimagi/commcare-hq/pull/34843/files#diff-a0373ab9ee40bfb0f09c48e38e9abf68f4d84f10c8d8336693ab9df2c65154a6R11). That seems a bit confusing too.

Maybe what can be done is adding another hidden input with `name="confirm"` to the confirmation form so everything stays in `POST`?

Reply:
Looking back at the original ticket i see:

> Can we implement some validation prior to importing cases that ensures rows are properly formatted, or at least warn the user that they are about to import X cases and give them a chance to cancel and check the excel file for mistakes?

It looks like the latter route was taken. That should be fine...but maybe we can have a threshold for the number of cases at which we present the alert? I'm also curious what the erroneous excel file looked like and asked for more info in the ticket

Reply:
Although I realized Amit and Graham are offline...whoops. @dannyroberts do you possibly remember the related incident to this task which caused a pillow fire?

Reply:
@biyeun do you still have concerns around a full additional step confirming the size of an import when attempting an import? Trying to understand the state of this PR, and whether we should break up the part that targets flagging abnormalities vs the part that displays an import size confirmation.